src/Integration/Resources/Strings.resx

```diff
@@ -405,4 +405,12 @@ Localization note: '[Don't warn me again]()' is a special markup used to transla
     <value>SonarQube project binding failed: The server does not have the minimum required version ({0}) of the C# plugin installed.</value>
     <comment>Error message indicating that the server does not have the required version of the C# plugin installed. {0} minimum required version</comment>
   </data>
+  <data name="FailedToUnpackAdditionalFiles" xml:space="preserve">
+    <value>Failed to unpack additional files required for analyzers</value>
```

Comment:
'required for analyzers': 'specified by the quality profile' might be better and more contextual in terms of the information the user gets from our extension in the output.